moped-database/metadata/databases/default/tables/public_moped_proj_funding.yaml

```diff
@@ -65,7 +67,6 @@ select_permissions:
         - funding_source_id
         - funding_status_id
         - is_deleted
-        - is_editable
```

Comment:
No longer needed. All records in the `moped_proj_funding` table should be editable just like they are today since we aren't mixing with immutable eCAPRIS data in the same table.